### DISCUSSION TITLE
lane choice

### DISCUSSION THREAD

**Discussion by AsphaltDragon68** [slyh/Cities2-TrafficLightsEnhancement#43] [Ideas]
Is it possible to chose which lanes a straight ahead or even the turn lanes go to. IE. ahead lane tend to go to the outside lane instead of the inside lane that is actually straight across the intersection.

**Comment by PsykotropyK** [comment]
I support that suggestion. An I would suggest to extend it the way [TM:PE](https://steamcommunity.com/sharedfiles/filedetails/?id=1637663252) works in Cities Skylines 1. 
The idea is that once a crossing node is selected the player have access to all the entry points from the crossing, and can then individualy connects them to the exit points. 
That's a great tool to ensure that the turning lane can access all lanes on the target road, instead of only one lane, which should greatly fluidify traffic as currently, turning left may have the cars ending it the left most lane, which proved problematic if next path is going right.

Attached are some screenshots taken from TM:PE : 
Selected crossing node, where you can see the crossing entry point : 
![image](https://github.com/slyh/Cities2-Various-Mods/assets/12467463/0f73e750-aa5b-4b75-8e81-8b974a8e8e95)

Individual mapping to the exit points from one entry point : 
![image](https://github.com/slyh/Cities2-Various-Mods/assets/12467463/604c4fca-dbff-43fe-9de2-4731e679e340)

Complete crossing mapping : 
![image](https://github.com/slyh/Cities2-Various-Mods/assets/12467463/5e19b821-1b27-4faf-b4f6-62f864638424)

**Reply by krzychu124** [reply]
Guys, I'm not dead 🤣 I'm working on TM:PE, just behind the scenes (will be released under different, simpler name).
Be patient otherwise this mod will become completely incompatible once I release mine (now there's only a little incompatibility).
